pmm-app/src/pmm-update-panel/components/UpdateButton/UpdateButton.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+import { CenteredButton } from 'pmm-update-panel/components';
+import * as styles from './UpdateButton.styles';
+
+interface UpdateButtonProps {
+  onClick: () => void;
+  nextVersion: string;
+  disabled?: boolean;
+}
+
+export const UpdateButton = ({ onClick, nextVersion, disabled = false }: UpdateButtonProps) => (
+  <CenteredButton disabled={disabled} className={styles.updateButton} onClick={onClick} icon="fa fa-download">
```

Comment:
Pls try to use grafana icons from '@grafana/ui' vs fontawesome.

Reply:
Same as [this one](https://github.com/percona/grafana-dashboards/pull/766/files/4def364973bdb9834088795560da738f6d7fca77#r440811552)